.studio/common

```diff
@@ -33,16 +33,17 @@ function compile_go_protobuf_component() {
   # Note: the actual generation is triggered by a call to protoc with the
   # parameter '--policy_out=...'. Only if the service's methods has the proper
   # annotations will the generated code not be a NO-OP.
-  setup_go_workspace
-  ( cd "${scaffolding_go_pkg_path}" || return 1
-    local proto_go_tools=(
-      github.com/chef/automate/components/automate-grpc/protoc-gen-policy
-      github.com/chef/automate/components/automate-grpc/protoc-gen-grpc-mock # generate mock grpc server implementations
-      github.com/chef/automate/components/automate-grpc/protoc-gen-a2-config
-    )
-    install_go_tool "${proto_go_tools[@]}"
-    GO_TOOL_VENDOR=true install_go_tool github.com/envoyproxy/protoc-gen-validate
+  local proto_go_tools=(
+    github.com/chef/automate/components/automate-grpc/protoc-gen-policy
+    github.com/chef/automate/components/automate-grpc/protoc-gen-grpc-mock # generate mock grpc server implementations
+    github.com/chef/automate/components/automate-grpc/protoc-gen-a2-config
   )
+  install_go_tool "${proto_go_tools[@]}"
+  install_go_tool github.com/envoyproxy/protoc-gen-validate
+  install_go_tool github.com/ckaznocha/protoc-gen-lint
```

Comment:
[nit] Can `protoc-gen-validate` and `protoc-gen-lint` be added to `proto_go_tools`?